content/pages/photo-policy.md

```diff
@@ -0,0 +1,6 @@
+We take pictures at events, but won't take your picture if you're
```

Comment:
Also add at the end of this paragraph:

"We can use these pictures for things like our social media profiles (Facebook, Twitter) and also for our publicity material like brochures."